src/natcap/invest/ui/stormwater.py

```diff
@@ -0,0 +1,100 @@
+from natcap.invest.ui import model, inputs
```

Comment:
I was under the impression it isn't necessary since we're using the default, UTF-8. [Here's](https://stackoverflow.com/questions/14083111/should-i-use-encoding-declaration-in-python-3) a relevant stackoverflow. Is this just a best-practice thing?

Reply:
Cool, if you're ok with it I will leave it out :)